main.css

```diff
@@ -0,0 +1,142 @@
+/* General reset for margin and padding */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+
+.hero_container {
+    display: flex;
+    flex-direction: row;  
+    padding: 6vw;  
+    gap: 1vw;
+}
+
+.left-col {
+    flex: 2;  
+    line-height: 1.2;
+    padding: 10px;
+}
+
+.right-col {
+    flex: 1; 
+    padding: 10px;
+}
+
+.pic{
+border-radius: 100%;
+border: 15px solid purple;
+object-fit: cover; 
+}
+
+.Header_text{
+ font-size: 70px;
+ font-family: inter;
+ font-weight: 700;
+ margin-bottom: 2vw;
+}
+
+.Sub_text{
+    font-size: 16px;
+    margin-bottom: 3vw;
+    width: 90%;
+}
+
+.cta_btn {
```

Comment:
Looking at this class alone, I have no idea what it's for. Class names should be descriptive enough to have meaning so that anyone reading them can understand why they are being used. I would have a better understanding of why this class is being used if it was called ".join-meetup-button" instead of ".cta_btn".

Overall, keep in mind everyone is working with the same files, so being specific with the way things are named will make it a whole lot easier to work in a collaborative setting.